openmc/filter.py

```diff
@@ -104,27 +104,6 @@ def __lt__(self, other):
     def __hash__(self):
         return hash(repr(self))
 
-    def __deepcopy__(self, memo):
-        existing = memo.get(id(self))
-
-        # If this is the first time we have tried to copy this object, create a copy
-        if existing is None:
-            clone = type(self).__new__(type(self))
-            clone._type = self.type
-            clone._bins = copy.deepcopy(self.bins, memo)
```

Comment:
Now in this case one *would* want a deep rather than shallow copy of the bins. If I understand correctly, the default implementation will actually deep copy all of the class attributes?